src/main/java/utils/filereaders/CsvFileReader.java

```diff
@@ -28,8 +28,14 @@ public List<String> readFile(File file) {
             }
         } catch (IOException exception) {
             LOGGER.error("An error occurred while reading the file: {}\n{}", file.getPath(), exception);
+            try {
```

Comment:
this logic is specific to the use case, not to every csv, move it to the configurations class